frappe/templates/includes/login/login.js

```diff
@@ -57,12 +57,14 @@ login.bind_events = function() {
 	});
 
 	$(".toggle-password").click(function() {
```

Comment:
Tip: You can also check out [.toggle](https://api.jquery.com/toggle-event/) function